compute_tools/src/compute.rs

```diff
@@ -180,6 +181,24 @@ impl TryFrom<ComputeSpec> for ParsedSpec {
     }
 }
 
+/// Spawns a command in the neon-postgres cgroup if it exists, otherwise just
+/// spawns it normally, returning a handle to it.
```

Comment:
comment is outdated? (spawns in neon-postgres if _autoscaling_, ignoring whether the cgroup exists)